src/bidiTab/bidiTab.ts

```diff
@@ -282,7 +285,11 @@ class BidiParserImpl implements BidiParser {
     return Parser.Script.parseDisownParams(params);
   }
   parseSendCommandParams(params: object): CDP.SendCommandParams {
-    return Parser.CDP.parseSendCommandParams(params);
+    return Parser.CDP.parseSendCommandParams(params) as {
+      cdpMethod: keyof ProtocolMapping.Commands;
+      cdpParams: object;
+      cdpSession?: any;
+    };
```

Comment:
Why not to change `SendCommandParams` accordingly?

Reply:
Done!